### ISSUE TITLE
Adding timer to task bar

### ISSUE THREAD

**Issue by annupriy** [SauveJeanLuc/coffee-tasks#136]
As discussed, We can add a timer icon to the taskbars, which when tapped will show how much time is left for the user to complete their task.

We would also need to ask the user for a set time duration when they create a task.

**Comment by mnosov622** [comment]
Hi @annupriy !

I would love to work on this issue.

Could you assign it to me, please ?

**Comment by annupriy** [comment]
> Hi @annupriy !
> 
> I would love to work on this issue.
> 
> Could you assign it to me, please ?

Hey @mnosov622! I am not the owner of this repository. @SauveJeanLuc and I discussed this issue in the previous thread and I created this issue to work on it. 

You can definitely find another issue to work on.

**Comment by SauveJeanLuc** [comment]
@annupriy what do you mean by the task bar for the app?
Do you mean this timer will apply for all tasks ?

**Comment by SahilSingh177** [comment]
> @annupriy what do you mean by the task bar for the app? Do you mean this timer will apply for all tasks ?

No, the timer will show the time remaining in the task whose deadline is the nearest. And as soon as it reaches the deadline an alert will be given.

**Comment by annupriy** [comment]
> @annupriy what do you mean by the task bar for the app? Do you mean this timer will apply for all tasks ?

No, I meant that a timer will be shown for the task whose deadline is the nearest and as soon as it reaches the deadline an alert will be given to the user.

**Comment by SauveJeanLuc** [comment]
Great,

> > @annupriy what do you mean by the task bar for the app? Do you mean this timer will apply for all tasks ?
> 
> No, I meant that a timer will be shown for the task whose deadline is the nearest and as soon as it reaches the deadline an alert will be given to the user.
Great, go for it.

**Comment by SauveJeanLuc** [comment]
@annupriy are you still working on this ?
